frontend/src/App.tsx

```diff
@@ -195,6 +204,12 @@ function appStateReducer(state: CoveyAppState, update: CoveyAppUpdate): CoveyApp
         nextState.nearbyPlayers = state.nearbyPlayers;
       }
       break;
+    case 'playerRaceSettings': 
```

Comment:
Maybe this should be called 'updatePlayerRaceSettings', and the other one as 'toggleRaceSettings' instead.  The name's a little confusing